packages/select/src/Select/Select.types.ts

```diff
@@ -35,7 +37,7 @@ export interface BaseSelectProps
       HTMLElementProps<'button', HTMLButtonElement>,
       'onChange' | 'onClick'
     >,
-    Omit<PopoverProps, 'active' | 'spacing'>,
+    Omit<PopoverProps, 'active' | 'dismissMode' | 'onToggle' | 'spacing'>,
```

Comment:
Not really sure what's causing the duplicated docs in SB. (the VSCode TSDoc highlights docs are fine, so not super pressing)
![Screenshot 2024-10-14 at 1 20 13 PM](https://github.com/user-attachments/assets/6b3dde28-869f-4cb4-85bd-3b435be6ecfe)